### ISSUE TITLE
Getting this error with the new translation update.

### ISSUE THREAD

**Issue by artistro08** [OFFLINE-GmbH/oc-mall-plugin#196]
I could not re-create this issue on the site, but I get this stacktrace anytime I want to change something in the backend: 

```
Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function concat() on null in /home/styleclo/public_html/styleclothingco/plugins/offline/mall/models/Variant.php:204
Stack trace:
#0 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php(438): OFFLINE\Mall\Models\Variant->getAllPropertyValuesAttribute(NULL)
#1 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php(340): Illuminate\Database\Eloquent\Model->mutateAttribute('all_property_va...', NULL)
#2 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(993): Illuminate\Database\Eloquent\Model->getAttributeValue('all_property_va...')
#3 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(957): October\Rain\Database\Model->getAttributeValue('all_property_va...')
#4 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/models/Variant.php(209): October\Rain\Database\Model->getAttribute('all_property_va...')
#5 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(1382): OFFLINE\Mall\Models\Variant->getAttribute('all_property_va...')
#6 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Extension/ExtendableTrait.php(360): Illuminate\Database\Eloquent\Model->__get('all_property_va...')
#7 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(617): October\Rain\Database\Model->extendableGet('all_property_va...')
#8 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/classes/index/VariantEntry.php(36): October\Rain\Database\Model->__get('all_property_va...')
#9 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/classes/observers/ProductObserver.php(34): OFFLINE\Mall\Classes\Index\VariantEntry->__construct(Object(OFFLINE\Mall\Models\Variant))
#10 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(87): OFFLINE\Mall\Classes\Observers\ProductObserver->created(Object(OFFLINE\Mall\Models\Product))
#11 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Support/Collection.php(339): OFFLINE\Mall\Console\ReindexProducts->OFFLINE\Mall\Console\{closure}(Object(OFFLINE\Mall\Models\Product), 0)
#12 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(89): Illuminate\Support\Collection->each(Object(Closure))
#13 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Concerns/BuildsQueries.php(39): OFFLINE\Mall\Console\ReindexProducts->OFFLINE\Mall\Console\{closure}(Object(October\Rain\Database\Collection), 1)
#14 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(90): Illuminate\Database\Eloquent\Builder->chunk(200, Object(Closure))
#15 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(38): OFFLINE\Mall\Console\ReindexProducts->reindex()
#16 [internal function]: OFFLINE\Mall\Console\ReindexProducts->handle(Object(OFFLINE\Mall\Classes\Index\Filebase))
#17 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(29): call_user_func_array(Array, Array)
#18 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#19 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(October\Rain\Foundation\Application), Array, Object(Closure))
#20 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(October\Rain\Foundation\Application), Array, Array, NULL)
#21 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Command.php(183): Illuminate\Container\Container->call(Array)
#22 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Command/Command.php(264): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#23 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Command.php(170): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#24 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(874): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#25 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(228): Symfony\Component\Console\Application->doRunCommand(Object(OFFLINE\Mall\Console\ReindexProducts), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#26 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(130): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#27 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Application.php(88): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#28 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(121): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#29 /home/styleclo/public_html/styleclothingco/artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#30 {main} [2019-04-28 07:39:56] production.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function concat() on null in /home/styleclo/public_html/styleclothingco/plugins/offline/mall/models/Variant.php:204 Stack trace: #0 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php(438): OFFLINE\Mall\Models\Variant->getAllPropertyValuesAttribute(NULL) #1 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php(340): Illuminate\Database\Eloquent\Model->mutateAttribute('all_property_va...', NULL) #2 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(993): Illuminate\Database\Eloquent\Model->getAttributeValue('all_property_va...') #3 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(957): October\Rain\Database\Model->getAttributeValue('all_property_va...') #4 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/models/Variant.php(209): October\Rain\Database\Model->getAttribute('all_property_va...') #5 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(1382): OFFLINE\Mall\Models\Variant->getAttribute('all_property_va...') #6 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Extension/ExtendableTrait.php(360): Illuminate\Database\Eloquent\Model->__get('all_property_va...') #7 /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Database/Model.php(617): October\Rain\Database\Model->extendableGet('all_property_va...') #8 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/classes/index/VariantEntry.php(36): October\Rain\Database\Model->__get('all_property_va...') #9 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/classes/observers/ProductObserver.php(34): OFFLINE\Mall\Classes\Index\VariantEntry->__construct(Object(OFFLINE\Mall\Models\Variant)) #10 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(87): OFFLINE\Mall\Classes\Observers\ProductObserver->created(Object(OFFLINE\Mall\Models\Product)) #11 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Support/Collection.php(339): OFFLINE\Mall\Console\ReindexProducts->OFFLINE\Mall\Console\{closure}(Object(OFFLINE\Mall\Models\Product), 0) #12 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(89): Illuminate\Support\Collection->each(Object(Closure)) #13 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Database/Concerns/BuildsQueries.php(39): OFFLINE\Mall\Console\ReindexProducts->OFFLINE\Mall\Console\{closure}(Object(October\Rain\Database\Collection), 1) #14 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(90): Illuminate\Database\Eloquent\Builder->chunk(200, Object(Closure)) #15 /home/styleclo/public_html/styleclothingco/plugins/offline/mall/console/ReindexProducts.php(38): OFFLINE\Mall\Console\ReindexProducts->reindex() #16 [internal function]: OFFLINE\Mall\Console\ReindexProducts->handle(Object(OFFLINE\Mall\Classes\Index\Filebase)) #17 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(29): call_user_func_array(Array, Array) #18 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() #19 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(October\Rain\Foundation\Application), Array, Object(Closure)) #20 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(October\Rain\Foundation\Application), Array, Array, NULL) #21 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Command.php(183): Illuminate\Container\Container->call(Array) #22 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Command/Command.php(264): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle)) #23 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Command.php(170): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle)) #24 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(874): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #25 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(228): Symfony\Component\Console\Application->doRunCommand(Object(OFFLINE\Mall\Console\ReindexProducts), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #26 /home/styleclo/public_html/styleclothingco/vendor/symfony/console/Application.php(130): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #27 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Console/Application.php(88): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #28 /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(121): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #29 /home/styleclo/public_html/styleclothingco/artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #30 {main} [] []
 
    /home/styleclo/public_html/styleclothingco/plugins/vojtasvoboda/bugsnag/vendor/mead-steve/mono-snag/src/BugsnagHandler.php:65 MeadSteve\MonoSnag\BugsnagHandler::write
    /home/styleclo/public_html/styleclothingco/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php:39 Monolog\Handler\AbstractProcessingHandler::handle
    /home/styleclo/public_html/styleclothingco/vendor/monolog/monolog/src/Monolog/Logger.php:344 Monolog\Logger::addRecord
    /home/styleclo/public_html/styleclothingco/vendor/monolog/monolog/src/Monolog/Logger.php:707 Monolog\Logger::error
    /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Log/Writer.php:203 Illuminate\Log\Writer::writeLog
    /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Log/Writer.php:114 Illuminate\Log\Writer::error
    /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:221 Illuminate\Support\Facades\Facade::__callStatic
    /home/styleclo/public_html/styleclothingco/vendor/october/rain/src/Foundation/Exception/Handler.php:50 October\Rain\Foundation\Exception\Handler::report
    /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:352 Illuminate\Foundation\Console\Kernel::reportException
    /home/styleclo/public_html/styleclothingco/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:131 Illuminate\Foundation\Console\Kernel::handle
    /home/styleclo/public_html/styleclothingco/artisan:35 [main]
```

**Comment by artistro08** [comment]
Looks like re-indexing and then updating fixed the issue

**Comment by tobias-kuendig** [comment]
Thank you for this bug report! This has been fixed in 2569899 and will be released later today.

**Comment by artistro08** [comment]
awesomeness. Just got the update
